apps/next-evans/lib/utils.ts

```diff
@@ -1,3 +1,7 @@
 export function isNotNullish<T>(x: T | null | undefined): x is T {
   return x !== null && x !== undefined;
 }
+
+export function inflect(str: string, count: number): string {
```

Comment:
i'm not sure this holds up for spanish too, and i think if we do pull over the other seo file, we won't need this, is my thought.